### ISSUE TITLE
Create an ongoing customer journey experience 

### ISSUE THREAD

**Issue by serejandmyself** [citizen-cosmos/Citizen-Cosmos#22]
I.E.: To learn how to launch a DAO, listen to episode X. To find out about market predictions...

- [ ] Create a list
- [ ] Publish to blog

**Comment by johnniecosmos** [comment]
What exactly is the task here?

**Comment by serejandmyself** [comment]
This is a future task. I will ping you up when I write it out in more detail. The general idea is to use `regular-world` instruments and apply them to communication. e

You probably heard about a customer or a client journey. i.e. [1](https://customerjourney.uk.com/customer-journey/what-is-customer-journey/)

The idea is to use the same method for newcomers into crypto via the info that the podcast guests provide. For example, the first episode teaches us that governance is important. Now, as a new comer, to me that's gibberish. Whta if there was some kind of an intro doc or game or whatever that would help you to learn all that and tell you in plain language where to go. i.e.

Governance is required to... Listen to Gavin from Figment network on this timestamp of this episode to learn why its important and read this to know how to do it. 

Long story short, Thats the general idea. But its not polished yet

**Comment by johnniecosmos** [comment]
Great sounds fun we will explore it when the time comes then, but i like the idea mate!

**Comment by serejandmyself** [comment]
Re-org the task

**Comment by serejandmyself** [comment]
#21
